bin/router/src/schema_state.rs

```diff
@@ -118,6 +125,8 @@ impl SchemaState {
         let supergraph_state = SupergraphState::new(&parsed_supergraph_sdl);
         let planner = Planner::new_from_supergraph(&parsed_supergraph_sdl)?;
         let metadata = planner.consumer_schema.schema_metadata();
+        // TODO: use supergraph
```

Comment:
TODO

Reply:
It's a draft :) You have no idea how many more `TODO:` comments I have locally still